src/auth/recaptcha-config.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright 2020 Google Inc.
```

Comment:
2022

Reply:
Moved  reCaptcha config to auth-config.  This issue does not applicable anymore.